src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java

```diff
@@ -506,9 +535,9 @@ private void copyLogs() {
 
                     String prefix = getLogPrefix(node);
                     if (prefix != null) {
-                        logger = new LogLinePrefixOutputFilter(listener.getLogger(), "[" + prefix + "] ");
+                        logger = new LogLinePrefixOutputFilter(getListener().getLogger(), "[" + prefix + "] ");
```

Comment:
Tons of merge conflicts with #27 coming up, sigh. Was the introduction of `getListener()` necessary to solve one of these bugs?